test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetVerifyTests.cs

```diff
@@ -141,5 +144,488 @@ public void Verify_MultipleSignedPackagesWithWildCardAndDetailedVerbosity_MixedR
                 }
             }
         }
+
+        [PlatformTheory(Platform.Windows, Platform.Linux)] // https://github.com/NuGet/Home/issues/11178
+        [InlineData("true", true)]
+        [InlineData("true", false)]
+        [InlineData("false", true)]
+        [InlineData("false", false)]
+        public void Verify_AuthorSignedPackage_WithAuthorItemTrustedCertificate_Succeeds(string trust, bool fingerprintOption)
```

Comment:
Why are assertions different between corresponding NuGet.exe and dotnet verify tests?  I expect them to be identical.

Reply:
For both `nuget verify` and `dotnet nuget verify` just reused existing tests in same file.
Addressed, I just made them look similar.